### ISSUE TITLE
Script

### ISSUE THREAD

**Issue by vulnz** [SethBrunwasser/Phishing-ML#1]
Hello, this info gathered by some script?
can you show it?

**Comment by SethBrunwasser** [comment]
Hi Vulnz,
The data was found online. I did not scrape the data but am interested in making a script to gather more data.

**Comment by vulnz** [comment]
nice. how to check website using this data?

**Comment by SethBrunwasser** [comment]
You mean how can this model be deployed to test websites live?
To be honest, I'm not sure. I'm sure there exists or you could make a script that uses web tools to scrape the features of a website.

**Comment by vulnz** [comment]
I mean that if I would like to check specific website for being phishing one.
./script.py -url test.com + using your data or some live data

**Comment by SethBrunwasser** [comment]
Yeah you would need to somehow get the features  of the website like HTTP_token, DNSRecord, etc.
